src/upchunk.ts

```diff
@@ -75,6 +94,10 @@ export class UpChunk  {
     this.offline = false;
     this.paused = false;
     this.success = false;
+    this.chunkHistory = [];
```

Comment:
I assume there's no risk of a meaningful memory leak from keeping this chunk metadata around for the entire upload? Just want to ensure we've done some napkin math to be sure.

Reply:
I tested this with a 760 megabyte upload, which averaged uploading in 12 chunks, and the entire chunk history was only 809 bytes.  I think the debugging capability alone more than makes up for the 800 byte increase in memory usage.

Reply:
Of course, at the minimum chunk size, this would obviously grow much much larger.  I'll go ahead and comment it out for now.

Reply:
Or just put a cap on its size. Then anytime you append to it you can check its size / truncate the list as appropriate.

Yeah, it was specifically for min chunk size that I was concerned, especially since those may be the least powerful clients.